### PR TITLE
Update blog for comments from community

### PR DESCRIPTION
**Pull request by kevin-ortega**
#2111 Updating blog because of comments received from PM.

### CODE REVIEW

**Comment by kevin-ortega** [comment]
image
image

https://blogs-draft-openlibertyio.mybluemix.net/blog/2022/04/01/cloud-native-liberty-buildpack.html

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 1]
```diff
@@ -94,17 +94,49 @@ Set a default builder:
...
+  --env BP_LIBERTY_FEATURES='jaxrs-2.1 jsonp-1.1 cdi-2.0 mpMetrics-3.0 mpConfig-2.0' \
+  --buildpack paketo-buildpacks/eclipse-openj9 --buildpack paketo-buildpacks/java myapp
 
 Your application is now transformed into an OCI image!
 
 == Now what? 
-With your OCI image, you can run your application locally with the `docker run` command:
+With your OCI image, you can run your application locally with the `docker run` command.  
 
+We need to provide a `server.xml` to the Liberty buildpack using a binding.
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
We need to provide a `server.xml` file to the Liberty buildpack by using a binding.  
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 2]
```diff
@@ -94,17 +94,49 @@ Set a default builder:
...
 
 Your application is now transformed into an OCI image!
 
 == Now what? 
-With your OCI image, you can run your application locally with the `docker run` command:
+With your OCI image, you can run your application locally with the `docker run` command.  
 
+We need to provide a `server.xml` to the Liberty buildpack using a binding.  
+
+* Create a directory named `bindings`, for example
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]:
```suggestion
* Create a directory named `bindings`, for example.
```

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc [hunk 3]
```diff
@@ -94,17 +94,49 @@ Set a default builder:
...
+* Create a file named server.xml in the `bindings` directory with the following content:
+[source]
+<server description="Intro to the Paketo Liberty buildpack">
+  <featureManager>
+      <!--feature>webProfile-8.0</feature -->
```
Comment (dmuelle) [inline posts/2022-04-01-cloud-native-liberty-buildpack.adoc]: it looks like the webProfile-8.0 feature is commented out- is that intentional?

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review- a few minor suggestions and a question

File: posts/2022-04-01-cloud-native-liberty-buildpack.adoc (hunk 3 above)
Comment (kevin-ortega) [reply posts/2022-04-01-cloud-native-liberty-buildpack.adoc]: Ah...crap.  I'll remove it.

**Review by dmuelle — APPROVED** [review]
LGTM- please also send these changes back to draft if you haven't already- thanks

**Comment by kevin-ortega** [comment]
> 

already done!  Thanks David!

**Pull request merged**